### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by fenglish**
Add tests for 
- Buttons.jsx
- Players.jsx
- TimeRemaining.jsx
- Timeline.jsx
- Marquee.jsx
- PlaybackRate.jsx
- Image.jsx

### CODE REVIEW

File: components/x-audio/src/components/__tests__/PlaybackRate.test.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	const subject = shallow(<PlaybackRate {...props}/>);
+
+	test('should have necessary elements', () => {
+		expect(subject).toMatchSnapshot();
+	});
+
+	test('should call onClick function with next playbackRate when it is clicked', () => {
+		subject.simulate('click');
+		expect(subject).toMatchSnapshot();
+		expect(props.onClick).toHaveBeenCalledWith(1.5);
```
[the remark this answers is not in the document]
Comment (benbarnett) [reply components/x-audio/src/components/__tests__/PlaybackRate.test.jsx]: @debugwand @fenglish FYI there is a test for this against the helper itself: https://github.com/Financial-Times/x-dash/blob/x-audio/components/x-audio/src/components/__tests__/playback-rates.test.js

File: components/x-audio/src/components/__tests__/Players.test.jsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	playBackRateButtonSelector,
+	imageSelector,
+	titleSelector,
+	seriesNameSelector,
+	timelineSelector
+} = require('../../../__tests__/test-helper');
+
+jest.mock('../classnames-helper', () => {
+	return (...classes) => classes.join(' ');
+});
```
Comment (benbarnett) [inline components/x-audio/src/components/__tests__/Players.test.jsx]: Feels like this mock is being used in most of the test files now, maybe worth extracting to a helper?

Comment (fenglish) [reply components/x-audio/src/components/__tests__/Players.test.jsx]: 👍

File: components/x-audio/src/components/__tests__/TimeRemaining.test.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+jest.mock('../classnames-helper', () => {
+	return (...classes) => classes.join(' ');
+});
+
+describe('TimeRemaining', () => {
+
+	const	props = {
+		currentTime: 60*5,
+		duration: 60*60 + 60*30 + 60*5
```
Comment (benbarnett) [inline components/x-audio/src/components/__tests__/TimeRemaining.test.jsx]: Is it possible to express these times in a named variable?

i.e.

```js
const FIVE_SECONDS = 60 * 5;
```

**Comment by benbarnett** [comment]
@fenglish a slightly more general comment (although up to you obviously!), when you have branches that are longer-living, like this one, personally I think it is ok to use merge commits to bring it inline with the base branch, rather than rebasing. And then, once the PR is approved and ready to merge, it can be rebased and the commits tidied up.

The main reason for this is that when reviewing the PRs, it makes it easier for the reviewer to see what has changed since they last looked at it. With a rebase and force push, the commits are essentially re-created so the reviewer sometimes has to read through the diff again to see what has changed since they last looked.

Does that make any sense?

**Comment by fenglish** [comment]
@benbarnett Thank you for your comment. I didn't know that I could use `git merge` for this situation. Now I understand what I have done. From next time, I will use `git merge` instead of `git rebase`.  Less confusion for reviewers :muscle: !! Thank you for taking time for me. I really applicate this comment.

**Pull request merged**